src/EnergyPlus/SingleDuct.cc

```diff
@@ -6015,6 +6015,48 @@ namespace SingleDuct {
         }
     }
 
+    void setATMixerSizingProperties(int const &inletATMixerIndex, int const &controlledZoneNum, int const &curZoneEqNum)
+    {
+        int airLoopIndex = // find air loop associated with ATMixer
+            DataZoneEquipment::ZoneEquipConfig(controlledZoneNum).InletNodeAirLoopNum(SingleDuct::SysATMixer(inletATMixerIndex).CtrlZoneInNodeIndex);
+        bool SizingDesRunThisAirSys = false;                               // Sizing:System object found flag
+        CheckThisAirSystemForSizing(airLoopIndex, SizingDesRunThisAirSys); // check for Sizing:System object
+        if (SizingDesRunThisAirSys) {
+            int sysSizIndex =
+                UtilityRoutines::FindItemInList(FinalSysSizing(airLoopIndex).AirPriLoopName, SysSizInput, &SystemSizingInputData::AirPriLoopName);
+            if (sysSizIndex == 0) sysSizIndex = 1; // use first when none applicable
+            // set ATMixer outlet air conditions in ZoneEqSizing array
+            ZoneEqSizing(curZoneEqNum).ATMixerVolFlow = SingleDuct::SysATMixer(inletATMixerIndex).DesignPrimaryAirVolRate;
+            // If air loop has coils use SA conditions (below), else if OA sys has coils then precool conditions, else OA conditions
+            if (DataAirSystems::PrimaryAirSystem(airLoopIndex).CentralHeatCoilExists) {
+                ZoneEqSizing(curZoneEqNum).ATMixerHeatPriDryBulb = SysSizInput(sysSizIndex).HeatSupTemp;
+                ZoneEqSizing(curZoneEqNum).ATMixerHeatPriHumRat = SysSizInput(sysSizIndex).HeatSupHumRat;
+            } else if (DataAirSystems::PrimaryAirSystem(airLoopIndex).NumOAHeatCoils > 0) {
+                ZoneEqSizing(curZoneEqNum).ATMixerHeatPriDryBulb = SysSizInput(sysSizIndex).PreheatTemp;
+                ZoneEqSizing(curZoneEqNum).ATMixerHeatPriHumRat = SysSizInput(sysSizIndex).PreheatHumRat;
```

Comment:
I think these are wrong. The Precool/heatTemp is at the outdoor air outlet prior to mixing so there is a mixing problem here just like in those other functions that sets heating temp/humrat. I'll wait for comments before making that change.